curator/patterns/post-comments.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Title: Post Comments
+ * Slug: curator/post-comments
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:post-comments-form /--></div>
+<!-- /wp:group -->
+
+<!-- wp:spacer {"height":"60px"} -->
+<div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"5px"}},"layout":{"type":"flex","flexWrap":"nowrap","orientation":"horizontal","justifyContent":"left"}} -->
+<div class="wp-block-group"><!-- wp:paragraph {"style":{"typography":{"textTransform":"capitalize"}},"textColor":"foreground","fontSize":"x-large"} -->
+<p class="has-foreground-color has-text-color has-x-large-font-size" style="text-transform:capitalize"><?php echo esc_html__( 'Comments', 'curator' ); ?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"textColor":"foreground","fontSize":"x-large"} -->
+<p class="has-foreground-color has-text-color has-x-large-font-size">(</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:post-comments-count {"textAlign":"center","textColor":"foreground","fontSize":"x-large"} /-->
+
+<!-- wp:paragraph {"textColor":"foreground","fontSize":"x-large"} -->
+<p class="has-foreground-color has-text-color has-x-large-font-size">)</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group --></div>
```

Comment:
Adjust block gap to `0` and add right padding to comments block.


| Current  | Expected |
| ------------- | ------------- |
| <img width="243" alt="image" src="https://user-images.githubusercontent.com/1935113/169810294-3fb94c39-974b-45bb-9580-37352027c9d1.png">  | <img width="243" alt="image" src="https://user-images.githubusercontent.com/1935113/169810213-c2738566-8a8e-4ef5-9888-3e21452becb6.png">  |